src/PowerShellEditorServices/Services/PowerShellContext/PowerShellContextService.cs

```diff
@@ -1040,28 +1040,6 @@ public async Task ExecuteScriptWithArgsAsync(string script, string arguments = n
 
             if (arguments != null)
             {
-                // Need to determine If the script string is a path to a script file.
-                string scriptAbsPath = string.Empty;
-                try
-                {
-                    // Assume we can only debug scripts from the FileSystem provider
-                    string workingDir = (await ExecuteCommandAsync<PathInfo>(
-                        new PSCommand()
-                            .AddCommand("Microsoft.PowerShell.Management\\Get-Location")
-                            .AddParameter("PSProvider", "FileSystem"),
-                            sendOutputToHost: false,
-                            sendErrorToHost: false).ConfigureAwait(false))
-                        .FirstOrDefault()
-                        .ProviderPath;
-
-                    workingDir = workingDir.TrimEnd(Path.DirectorySeparatorChar);
-                    scriptAbsPath = workingDir + Path.DirectorySeparatorChar + script;
-                }
-                catch (System.Management.Automation.DriveNotFoundException e)
-                {
-                    this.logger.LogHandledException("Could not determine current filesystem location", e);
-                }
-
```

Comment:
What's the explanation here?

Reply:
https://github.com/PowerShell/PowerShellEditorServices/pull/1540/commits/ad28c88d738cd178bd61dba400a65ba2fb5a64af